cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -662,10 +662,13 @@ export function changeFrameAsync(
 
             // commit the latest job frame to local storage
             localStorage.setItem(`Job_${job.id}_frame`, `${toFrame}`);
-            await job.logger.log(LogType.changeFrame, {
+            const log = await job.logger.log(LogType.changeFrame, {
                 from: frame,
                 to: toFrame,
-            });
+                step: toFrame - frame,
+                count: 1,
+            }, true);
+            log.close();
```

Comment:
It would be more correct to close this log after canvas got updated with a new image to compute correct duration. 
In case of multiple change frame events it will work almost as expected, but in case with one, duration will be about zero, what is not true